### PR TITLE
Lovecraftian Last Week in MIQ

### PR DESCRIPTION
**Pull request by d-m-u**
Lovecraft + Choose Your Own Adventure! Enjoy.

### CODE REVIEW

**Comment by JPrause** [comment]
@imtayadeway @syncrou @jrafanie can you review the next blog entry.
and Thanks @d-m-u for being willing to continue writing.

**Comment by JPrause** [comment]
It's like a rabbit hole,....just when I think I'm at the bottom,...poof,...another hole pops up.

**Comment by JPrause** [comment]
This isn't a blog entry,..it's an adventure. 👍

File: site/_posts/2017-05-01-llw-pause.html.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+title: "Lovecraftian Last Week in ManageIQ"
+author: d-m-u
+date: 2017-05-01
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+You stop to take the briefest of glances at gitter. It seems as though the latest release name has been getting some nontrivial attention as of late. Jason had attempted to warn the masses about the dangers, about the placid island of ignorance in the bay of benightedness on which we dwell in the midst of black seas of boundlessness. He had pleaded to no avail that it was not meant that we should voyage far. And the masses had not listened, nay, instead, he found himself shuddering, having to once again laboriously type Gaprindashvili, a name that chills his soul when he think of it and maddens him when he dreams of it. And so it was that Jason ended up cursed by the Gods of Very Long Names, who turn into [The Gods Who Say… ekki-ekki-ekki-ekki-ptang-zoom-boing-mrowr!][gods]
+
```
Comment (syncrou) [inline site/_posts/2017-05-01-llw-pause.html.md]: Ha!  - Quite poetic!

File: site/_posts/2017-05-01-llw-grimoire.html.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
...g, grim, singular giggling, that makes you shudder uncontrollably. No description will suffice, save to make you, dear reader, find such words the stuff of cheap imposters or madness. But you must be warned. The air in front of you grows cold and for an instant, you imagine that you can see beyond this frail world. It is a fact that the words you uttered so foolishly had wrought eternal havoc. Whether any could have lived through it is gravely open to doubt. The giggling grows in horrible immensity. You are seized suddenly by gigantic talons attached to a creature so horrible you dare not describe it in any significant detail. Red, dripping, a blob of pulsing jelly, with uncountable tentacled trunks, a monster from the stars which became more clearly defined as it continued to drain you of your blood. And in an instant, it was gone, the ghastly giggling still ringing in your ears as you fall to the floor, whispering, _but I [updated orders to derive action from order services product features][2]!_ You expire after incessant, pleading cries to be saved from some escaped denizens of hell. Even [scopes + vms][3] cannot save you. Death comes unmercifully in a way which cannot be told.
```
Comment (syncrou) [inline site/_posts/2017-05-01-llw-grimoire.html.md]: Is `[2]` pointing where you expected it too? - If it is maybe I missed the context.

File: site/_posts/2017-05-01-llw-review.html.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+The author, mysteriously enough, does not answer your desperate and repeated entreaties for help, and so you turn to the nearest warm body, the unfortunate LJ. As you describe the comment in no uncertain terms, he goes silent. Horrified, his face draining eerily of all color, he backs away slowly from your monitor, an anguished look on his face, muttering once to himself, “In his house at R’lyeh dead Cthulhu waits dreaming.” Little do you know that poor LJ has had some recent nightmares of his own: he has been recently haunted by the dreaded override_gem, and even the valiant attempts at [updating deps to latest][2] will not soothe his shaken soul.
```
Comment (syncrou) [inline site/_posts/2017-05-01-llw-review.html.md]: Is `[2]` the correct PR in reference to updating deps to latest

File: site/_posts/2017-05-01-llw-pause.html.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+In fits of madness, shrieking and waving your arms frantically, sometimes lapsing into an inane titter or a whisper about “the monsters from the angles of time”, you start liberally apply tape to all the corners of what few rooms you can still stand to inhabit, muttering about the evils of corners, covering mirrors and windows, starting erratically at dogs, and avoiding smoke. It is lucky for you that you are not even slightly more imaginative. Even as things are, your mind bends ever so infinitesimally. You go violently insane. Even the [use of correct model class for saving OrchestrationTemplate][2] or the [removal of jasmine from Gemfile and its subsequent move to the ui][3] cannot help you redeem your soul.
```
Comment (imtayadeway) [inline site/_posts/2017-05-01-llw-pause.html.md]: Possibly "you start liberally to apply tape"?

Comment (d-m-u) [inline site/_posts/2017-05-01-llw-pause.html.md]: Naw, starting: to react with a sudden brief involuntary movement.

**Comment by imtayadeway** [comment]
@d-m-u superb! :heart_eyes: 

Just one thing....have you given any thought to how you want these separate threads to appear on the [index page](http://manageiq.org/blog/) ? Perhaps you could hack them into a certain order by way of published date? Or it would be great if we only listed the first entry, but the others were still clickable. @hayesr any ideas?

File: site/_posts/2017-05-01-llw-grimoire.html.md (hunk 2 above)
Comment (d-m-u) [inline site/_posts/2017-05-01-llw-grimoire.html.md]: It's right. I don't there there was any context, I just wanted to use it somewhere and there seemed like a fine place.

**Comment by d-m-u** [comment]
I think we should only list the first entry. Can we do that?

**Comment by JPrause** [comment]
@imtayadeway that's a good point! Perhaps a separate folder where the 3 supporting entries could be merged. Then change the links to point there. Just thinking out loud.

**Comment by d-m-u** [comment]
@JPrause Do I follow the naming convention of having an underscore before the folder name?

**Comment by JPrause** [comment]
@d-m-u I think @hayesr should answer that question.

**Comment by hayesr** [comment]
Looking into it, I think we can override the excerpt for the index.

**Comment by JPrause** [comment]
@d-m-u did you have any further edits? If not, we'll see if @cybette can merge this asap.

**Comment by d-m-u** [comment]
Ready to go!

**Comment by JPrause** [comment]
Thanks @d-m-u 
@cybette can you review and merge when ready.

**Pull request merged**